src/EditorFeatures/CSharpTest/Diagnostics/SimplifyTypeNames/SimplifyTypeNamesTests.cs

```diff
@@ -3755,7 +3755,6 @@ void M()
     }
 }",
                 options: Option(CodeStyleOptions.QualifyPropertyAccess, false, NotificationOption.Warning),
-                diagnosticCount: 1,
```

Comment:
@dpoeschl  would you like to answer this? I know we talked about the diagnosticCount assert not actually checking anything but I can't remember the exact reason why... can the count ever be more than 1?